### PR TITLE
docs(authz/apps): Explain Execute permission for apps

### PR DESCRIPTION
**Pull request by dibyom**
Part of https://github.com/spinnaker/spinnaker/issues/3554

### CODE REVIEW

File: setup/security/authorization/index.md [hunk 1]
```diff
@@ -92,11 +92,21 @@ hal config provider $PROVIDER account edit $ACCOUNT \
...
 This method does not distinguish between `READ` and `WRITE` - users with access will have both.
 
 
 ### Applications
+Before Spinnaker 1.14, there were two types of restrictions to an application `READ` and `WRITE`.
+In the 1.14 release, a new permission type called `EXECUTE` was added. For any new applications,
+the permission required to trigger pipelines changes from groups with READ access to those with
+EXECUTE access.
+
+To maintain backwards compatibility for existing applications, groups with READ access will implicitly
```
Comment (dorbin) [inline setup/security/authorization/index.md]: s/backwards/backward/

File: setup/security/authorization/index.md [hunk 2]
```diff
@@ -92,11 +92,21 @@ hal config provider $PROVIDER account edit $ACCOUNT \
...
+get EXECUTE access. To change this behavior for a specific application, modify the application config
+in the UI to explicitly add EXECUTE permissions to a group. You can also flip the default behavior
```
Comment (dorbin) [inline setup/security/authorization/index.md]: Can you add a screen shot of the UI here?

Comment (dorbin) [reply setup/security/authorization/index.md]: Oh actually, now I see there's a screenshot below. Does it want to be updated?

But I'd move the `fiat.executFallback` config sentence to after that screen.

**Review by dorbin — APPROVED** [review]

**Comment by bpowell** [comment]
Thanks @dibyom !

File: setup/security/authorization/index.md (hunk 1 above)
Comment (dibyom) [reply setup/security/authorization/index.md]: done!

File: setup/security/authorization/index.md (hunk 2 above)
Comment (dibyom) [reply setup/security/authorization/index.md]: Added a new screenshot, and turned the two migration options into a list. The config sentence is below the screenshot

**Pull request merged**